### ISSUE TITLE
[Issue] something broke!

### ISSUE THREAD

**Issue by OM3GA6400** [AnOpenSauceDev/Methane-mod#112]
# 🔥 Something broke!

## Description of what happened

biome partical weather, methane

_it is recommended to use [GitHub Gists](https://gist.github.com/) when uploading large logfiles._

3.8.2-1.21

rain just goes through ceilings

**Comment by AnOpenSauceDev** [comment]
That's due to Methane's particle optimization. This _is_ working as intended, but I'm not sure how to tell the user that particle collision is quite CPU intense before switching collision back to vanilla.

**Comment by OM3GA6400** [comment]
is there an option to turn particle collision on?

**Comment by AnOpenSauceDev** [comment]
Not really, it does if it detects a mod that can cause a crash, but I never really put it into the config.

**Comment by OM3GA6400** [comment]
is there any way you can add a way to individually toggle the options via mod menu?

**Comment by om3ga6400** [comment]
@AnOpenSauceDev

**Comment by AnOpenSauceDev** [comment]
There isn't, but you could fork and remove the mixin reference from the mixin file/use a mixin plugin to disable it. Currently I'm not able to work on the mod right now, so I can't add an option to the config.
